HISTORY.rst

```diff
@@ -0,0 +1,15 @@
+.. :changelog:
+
+
+Release History
+---------------
+
+
+0.21.2 (2016-04-18)
++++++++++++++++++++
+
+
+* Switched from `django.utils.importlib` to Python's standard `importlib` to
```

Comment:
Looks like something is messed up in github here. Looking at both `master` and this branch, the `84d6cb9` commit is in both so shouldn't actually show up in here 😕 .

![screen shot 2016-05-25 at 4 35 42 pm](https://cloud.githubusercontent.com/assets/1236629/15559403/d5529998-2296-11e6-81ad-c6ccbaad9b71.png)